src/main/java/barcus/tasklist/TaskList.java

```diff
@@ -0,0 +1,119 @@
+package barcus.tasklist;
+
+import barcus.task.Task;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Class to encapsulate the list of tasks
+ */
+public class TaskList {
+    private ArrayList<Task> tasks;
```

Comment:
Just a minor preference but maybe rename to taskList or listOfTasks to be a bit more descriptive, otherwise no issues at all.